govtool/frontend/vite.config.ts

```diff
@@ -52,6 +52,11 @@ const vitestConfig = defineVitestConfig({
     globals: true,
     environment: "jsdom",
     reporters: "verbose",
+    coverage: {
+      provider: "v8",
+      reporter: ["html"],
```

Comment:
Let's also create junit reports too, as it's the most recognized format for import/export operations.